### PR TITLE
Fix style of extensions on left and right pane

### LINKED ISSUE #550: Code Snippet right panel style need fix
When switching code snippet panel to open on the right side, this is how it looks:
image

Compared to the default left panel:
image

### PR DESCRIPTION
**Pull request by karlaspuldaro**
Fixes #550 
Screenshot with changes:
image

Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: packages/pipeline-editor/style/index.css [hunk 1]
```diff
@@ -81,12 +81,12 @@
...
 .lm-Widget #jp-left-stack {
-  padding-left: 8px;
+  padding-left: unset;
 }
 
 .lm-Widget #jp-right-stack {
-  margin-top: 25px;
-  margin-right: 5px;
+  margin-top: 0px;
+  margin-right: 0px;
```
Comment (ajbozarth) [inline packages/pipeline-editor/style/index.css:89]: if this is unnecessary we should just remove it instead

Comment (karlaspuldaro) [reply packages/pipeline-editor/style/index.css:89]: I agree. Still looking at how the deletion can be picked up, as I tried a clean build and clearing browser cache as suggested but it didn't help

Comment (ajbozarth) [reply packages/pipeline-editor/style/index.css:89]: I traced back to the original commit https://github.com/elyra-ai/elyra/commit/b842f49ae095df4ef98b11f88d07e1a52356b259 and I'm not sure these ever were needed

Comment (ajbozarth) [reply packages/pipeline-editor/style/index.css:89]: I just realized after noting that the pr number for that commit didn't add up that the commit is from when the button was still its own repo. Back then it had it's own ui that opened in the right sidebar. We must have not removed the css when we removed that part of the ui

Comment (ajbozarth) [reply packages/pipeline-editor/style/index.css:89]: I tested with these deleted and it looks great.

**Comment by karlaspuldaro** [comment]
Thx @ajbozarth also for helping in getting my clean script to properly work, that was the issue. 
Only when overriding css selections worked for me, because when deleting those lines, my browser would pick up an old css file from notebook-scheduler-extension, which was still installed in my env

**Pull request merged**